internal/sirius/get_firm_deputies.go

```diff
@@ -50,7 +58,18 @@ type FirmDeputy struct {
 	ActiveClientsCount   int
 	ExecutiveCaseManager string
 	OrganisationName     string
-	Town                 string
+	Town                 string  
+	ReviewDate           string
+	MarkedAsLabel        string
+	MarkedAsClass        string
+	AssuranceType        string
+}
+
+
+
+type RefData struct {
```

Comment:
We made a generic one of these that you should be able to access across all files, it's `HandleLabel` in get_deputy_details.